src/views/Opportunities/OpportunityComponents/OpportunityForm.jsx

```diff
@@ -223,6 +224,19 @@ const OpportunityForm = (props) => {
     });
   };
 
+    if (props.id) {
+      const searchedemployed = employerOptions.find((option) => option.label === props.employer.name);
+      if (!searchedemployed) {
+        const newvalue = {
+          key: props.employer.name,
+          label: props.employer.name,
+          value: Number(props.employer.id),
+        };
+        setNewValues(newvalue);
+        employerOptions.push(newValues);
```

Comment:
why do we need a reactive variable? Can't we directly do this?
```suggestion
        employerOptions.push(newValue);
```

Reply:
ok, sir i do it direct way.